### ISSUE TITLE
isort looks throw off by comments

### ISSUE THREAD

**Issue by frenck** [timothycrosley/isort#1264]
I've tried to upgrade the Home Assistant project today to isort 5.0.3, unfortunately, that seems to be not happening.

I've used the following config in `setup.cfg`:

```
[isort]
profile=black
indent = "    "
force_sort_within_sections = true
sections = FUTURE,STDLIB,INBETWEENS,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
default_section = THIRDPARTY
known_first_party = homeassistant,tests
forced_separate = tests
combine_as_imports = true
```

After I run isort everything seems fine. However, when I run it again, it modifies some files again, and again, and again...

After a couple of runs, I end up with files like this:

![image](https://user-images.githubusercontent.com/195327/86529123-53ef4c80-beae-11ea-974e-8784fdf54710.png)

**Comment by timothycrosley** [comment]
Thanks for reporting! This is fixed in the isort 5.0.4 patch release, with a regression test added to ensure it doesn't happen again

**Comment by frenck** [comment]
Thanks for the quick response on this @timothycrosley, that is appreciated 👍
